server/start-server.sh

```diff
@@ -4,11 +4,11 @@ set -o pipefail # if any code doesn't return 0, exit the script
 
 
 function setup_server() {
-  python manage.py makemigrations
-  python manage.py migrate
+  python server/manage.py makemigrations
```

Comment:
What happens when you don't use the `server/` path?

Reply:
The python interpreter in the virtual environment will not be able to locate the manage.py file